src/neuron/container/soa_container.hpp

```diff
@@ -5,6 +5,7 @@
 #include "neuron/container/generic_data_handle.hpp"
 #include "neuron/container/soa_identifier.hpp"
 
+#include <algorithm>
```

Comment:
```suggestion
#include <algorithm>  // std::transform
```
Would be nice to know _why_ headers are included, also allows to check if includes can be removed again.